oasislmf/model_preparation/sum_inputs.py

```diff
@@ -142,17 +103,12 @@ def merge_oed_to_mapping(summary_map_df, exposure_df, oed_column_set):
     :rtype: list
     """
 
-    # Guard check that set(column_set) < set(exposure_col_df.columns.to_list())
-    # --> OasisExecption if not
-
-    # Add or split `merge_oed_on_accounts`
     column_set = [c.lower() for c in oed_column_set]
     exposure_col_df = exposure_df[column_set]
```

Comment:
As discussed, it is better here to use

    exposure_col_df = exposure_df.loc[:, column_set]